app/services/contacts_service.rb

```diff
@@ -2,6 +2,7 @@
 
 class ContactsService
   CACHE_OPTIONS = { expires_in: 15.minutes, race_condition_ttl: 30.seconds }.freeze
+
```

Comment:
Did you mean to add a new line here?

Reply:
It's vital to the PR ;-). No, it was just to get a PR up to verify the issue. It's gone now, so I'll merge...